jbpm-wb-executor-service/jbpm-wb-executor-service-client/src/main/java/org/jbpm/workbench/es/client/editors/errorlist/ExecutionErrorListPresenter.java

```diff
@@ -320,9 +320,9 @@ public void setupAdvancedSearchView() {
 
         final Map<String, String> acks = new HashMap<>();
         final org.jbpm.workbench.common.client.resources.i18n.Constants constants = org.jbpm.workbench.common.client.resources.i18n.Constants.INSTANCE;
-        acks.put(String.valueOf(Boolean.TRUE),
+        acks.put(String.valueOf(new Short("1")),
```

Comment:
Do we still need to convert to String? That was valid for the boolean conversion before but I guess we can simply use the number now.